### PR TITLE
Add TeamsAIPolicy cmdlets public docs

### PR DESCRIPTION
**Pull request by Andy447**
Add new TeamsAIPolicy cmdlets public docs for 5 cmdlets

### CODE REVIEW

**Comment by Andy447** [comment]
@microsoft-github-policy-service agree company="{Microsoft}"

File: teams/teams-ps/teams/Grant-CsTeamsAIPolicy.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+```
+
+### GrantToTenant
+```
+Grant-CsTeamsAIPolicy [[-PolicyName] <String>] [-Global] [-Force] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet applies an instance of the Teams media connectivity policy to users or groups in a tenant.
```
Comment (shivtorov) [inline teams/teams-ps/teams/Grant-CsTeamsAIPolicy.md:42]:
```suggestion
This cmdlet applies an instance of the Teams AI policy to users or groups in a tenant.
```

File: teams/teams-ps/teams/Grant-CsTeamsAIPolicy.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+online version: https://learn.microsoft.com/powershell/module/teams/Grant-CsTeamsAIPolicy
+schema: 2.0.0
+author: Andy447
+ms.author: andywang
+---
+
+# Grant-CsTeamsAIPolicy
+
+## SYNOPSIS
+This cmdlet applies an instance of the Teams media connectivity policy to users or groups in a tenant.
```
Comment (shivtorov) [inline teams/teams-ps/teams/Grant-CsTeamsAIPolicy.md:15]:
```suggestion
This cmdlet applies an instance of the Teams AI policy to users or groups in a tenant.
```

**Comment by Andy447** [comment]
@shivtorov Can you please review this PR? Thank you!

**Comment by shivtorov** [comment]
> [Andy447](/Andy447)
@Andy447, I'm not a Microsoft employee, just a random contributor.

File: teams/teams-ps/teams/Get-CsTeamsAIPolicy.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+This cmdlet retrieves all Teams AI policies for current tenant.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-CsTeamsAIPolicy
+```
+
+retrieves Teams AI policies, shows "EnrollFace" and "EnrollVoice" values.
```
Comment (michbrown-png) [inline teams/teams-ps/teams/Get-CsTeamsAIPolicy.md:35]: Please capitalize the first letter of the word "Retrieves". To confirm, this cmdlet retrieves all Teams AI policies created for this tenant and shows what the EnrollFace and EnrollVoice values are for each policy?

File: teams/teams-ps/teams/Set-CsTeamsAIPolicy.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+## SYNTAX
+
+```
+Set-CsTeamsAIPolicy -Identity <string> -EnrollVoice <Enabled/Disabled>
+```
+
+## DESCRIPTION
+
+This cmdlet Set Teams AI policy EnrollFace/EnrollVoice value for current tenant. The value can be "Enabled" or "Disabled"
```
Comment (michbrown-png) [inline teams/teams-ps/teams/Set-CsTeamsAIPolicy.md:26]: Please make this description a little more clear. It is a bit ambiguous.

Recommended Edits:
This cmdlet sets the Teams AI policy EnrollFace and EnrollVoice value for the tenant. The values of EnrollFace and EnrollVoice can be set to "Enabled" or "Disabled"

**Review by michbrown-png — CHANGES_REQUESTED** [review]

**Comment by michbrown-png** [comment]
@Andy447 - Please make the suggested changes

**Pull request merged**